pkg/apis/resource/validation/validation_resourceclaim_test.go

```diff
@@ -360,6 +364,8 @@ func TestValidateClaimUpdate(t *testing.T) {
 }
 
 func TestValidateClaimStatusUpdate(t *testing.T) {
+	featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.DRAResourceClaimDeviceStatus, true)
```

Comment:
usually this will be an input to test with the feature enable and disable

Reply:
I replicated the tests cases for when this features is enabled and disabled, thank you.